tools/appliance/systemvmtemplate/scripts/configure_grub.sh

```diff
@@ -34,7 +34,7 @@ function configure_grub() {
 GRUB_DEFAULT=0
 GRUB_TIMEOUT=0
 GRUB_DISTRIBUTOR=Debian
-GRUB_CMDLINE_LINUX_DEFAULT="quiet"
+GRUB_CMDLINE_LINUX_DEFAULT="quiet fsck.mode=force fsck.repair=yes"
```

Comment:
these systemvm templates changes are merged with the PR https://github.com/apache/cloudstack/pull/9625.